tutorials/interoperability/xcm-transfers/from-relaychain-to-parachain.md

```diff
@@ -0,0 +1,154 @@
+
+        ```bash
+        X2mE9hCGX771c3zzV6tPa8U2cDz4U4zkqUdmBrQn83M3cm7
+        ```
+
+    3. **assets** - lists the assets to be withdrawn, including those designated for fee payment on the destination chain
+    4. **feeAssetItem** - indicates the index of the asset within the assets list to be used for paying fees
+    5. **weightLimit** - specifies the weight limit, if applicable, for the fee payment on the remote chain
+    6. Click on the **Submit Transaction** button to send the transaction
+
+        ![](/images/tutorials/interoperability/xcm-transfers/from-relaychain-to-parachain/from-relaychain-to-parachain-05.webp)
+
+After submitting the transaction, verify that the `xcmPallet.FeesPaid` and `xcmPallet.Sent` events have been emitted:
+
+![](/images/tutorials/interoperability/xcm-transfers/from-relaychain-to-parachain/from-relaychain-to-parachain-06.webp)
+
+### From the Parachain Perspective
+
+After submitting the transaction from the relay chain, confirm its success by checking the parachain's events. Look for the `assets.Issued` event, which verifies that the assets have been issued to the destination as expected:
+
+![](/images/tutorials/interoperability/xcm-transfers/from-relaychain-to-parachain/from-relaychain-to-parachain-07.webp)
+
+## Using PAPI
+
+To programmatically execute the reserve-backed asset transfer between the relay chain and the parachain, you can use [Polkadot API (PAPI)](/develop/toolkit/api-libraries/papi.md){target=\_blank}. PAPI is a robust toolkit that simplifies interactions with Polkadot-based chains. For this project, you'll first need to set up your environment, install necessary dependencies, and create a script to handle the transfer process.
+
+1. Start by creating a folder for your project
+   ```bash
+   mkdir reserve-backed-asset-transfer
+   cd reserve-backed-asset
+   ```
+
+2. Initialize a Node.js project and install the required dependencies. Execute the following commands:
+
+    ```bash
+    npm init
+    npm install polkadot-api @polkadot-labs/hdkd @polkadot-labs/hdkd-helpers
+    ```
+
+3. To enable static, type-safe APIs for interacting with the Polkadot and Astar chains, add their metadata to your project using PAPI
+
+    ```bash
+    npx papi add dot -n polkadot
+    npx papi add astar -w wss://rpc.astar.network
+    ```
+
+    !!! note 
+        - `dot` and `astar` are arbitrary names you assign to the chains, allowing you to access their metadata information
+        - The first command uses the well-known Polkadot chain, while the second connects to the Astar chain using its WebSocket endpoint
+
+4. Create a `index.js` file and insert the following code to configure the clients and handle the asset transfer
+
+    ```js
+    --8<-- 'code/tutorials/interoperability/xcm-transfers/from-relaychain-to-parachain/reserve-backed-transfer.js'
+    ```
+
```

Comment:
Add a note after the script that mentions that if they want to use this script for real-world blockchains, they have to change the ws to the real one, use a proper account instead of Alice and make sure they have enough money to pay fees